lib/puppet/defaults.rb

```diff
@@ -1570,7 +1570,10 @@ def self.default_diffargs
     :pluginsync => {
       :default    => true,
       :type       => :boolean,
-      :desc       => "Whether plugins should be synced with the central server.",
+      :desc       => "Whether plugins should be synced with the central server. This setting is
```

Comment:
@jtappa this language make sense to you? Related to https://tickets.puppetlabs.com/browse/PUP-5497.